src/Evolution/Systems/CurvedScalarWave/Worldtube/AccelerationTermsOrder0.cpp

```diff
@@ -0,0 +1,292 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/PunctureField.hpp"
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/DynamicBuffer.hpp"
+#include "DataStructures/Tags/TempTensor.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace CurvedScalarWave::Worldtube {
+
+void acceleration_terms_0(
+    gsl::not_null<Variables<tmpl::list<
+        CurvedScalarWave::Tags::Psi, ::Tags::dt<CurvedScalarWave::Tags::Psi>,
+        ::Tags::deriv<CurvedScalarWave::Tags::Psi, tmpl::size_t<3>,
+                      Frame::Inertial>>>*>
+        result,
+    const tnsr::I<DataVector, 3, Frame::Inertial>& centered_coords,
+    const tnsr::I<double, 3>& particle_position,
+    const tnsr::I<double, 3>& particle_velocity,
+    const tnsr::I<double, 3>& particle_acceleration, const double ft,
+    const double fx, const double fy, const double dt_ft, const double dt_fx,
+    const double dt_fy, const double bh_mass) {
+  const size_t grid_size = get<0>(centered_coords).size();
+  result->initialize(grid_size);
+  const double xp = particle_position[0];
+  const double yp = particle_position[1];
+  const double xpdot = particle_velocity[0];
+  const double ypdot = particle_velocity[1];
+  const double xpddot = particle_acceleration[0];
+  const double ypddot = particle_acceleration[1];
+  const double rp = get(magnitude(particle_position));
+  const double rpdot = (xp * xpdot + yp * ypdot) / rp;
+
+  const auto& Dx = get<0>(centered_coords);
+  const auto& Dy = get<1>(centered_coords);
+  const auto& z = get<2>(centered_coords);
+
+  const double M = bh_mass;
+
+  DynamicBuffer<DataVector> temps(56, grid_size);
```

Comment:
I think we want to have some guidelines regarding auto-generated code since it's not always the most optimal, just so we aren't baffled in 2 years when we look back at this. Could you

1. Add a comment here saying that the following code was auto-generated using Mathematica (or Sympy, or whatever you used)
2. If you have the code that generated this in a public repo/place, could you also add a comment pointing to it here? This way we can reproduce what you did.
3. Take a cursory look at the auto-generated code and if you notice any quick/simple optimizations that can be made, make those and add a comment where you did so we can note the difference. (I tried to do a couple below)